choreographer/src/test/resources/application.properties

```diff
@@ -32,10 +32,13 @@ log_all_request_and_response=false
 sr_address=127.0.0.1
 sr_port=8443
 
-# If true, address detection from HttpServletRequest will be performed at the time of executor registration/unregistration
-use_network_address_detector=false
-# Comma separated list representing the possible proxy servers like load-balancer etc... before Choreographer
-filter_proxy_addresses=
```

Comment:
Ah, so I've put these only into the tests at the time of exeutor resgistration implementation.